draft-frost-rats-eat-collection.md

```diff
@@ -161,6 +166,37 @@ In the registry {{IANA.cbor-tags}}, IANA is requested to allocate the tag in {{t
 | TBD399 | map | EAT Collection {{&SELF}} |
 {: #tbl-eat-collection align="left" title="EAT Collection"}
 
+## Media Type Registration
+{: #sec-iana-media-types}
+
+IANA is requested to register the "application/eat-collection" media type {{!RFC2046}} in the "Media Types" registry {{IANA-MediaTypes}} in the manner described in RFC 6838 {{!RFC6838}}, which can be used to indicate that the content is an EAT collection.
+
+* Type name: application
+* Subtype name: eat-collection
+* Required parameters: n/a
+* Optional parameters: n/a
+* Encoding considerations: binary
+* Security considerations: See the Security Considerations section
+  of {{&SELF}}
+* Interoperability considerations: n/a
+* Published specification: {{&SELF}}
+* Applications that use this media type: Attesters and Relying Parties sending
+  EAT collection attestation tokens over HTTP(S) and other transports.
```

Comment:
```suggestion
* Applications that use this media type: Attesters and Relying Parties sending
  EAT collection attestation evidence over HTTP(S) and other transports.
```

Reply:
It is a well formed token, and not a raw evidence, so I kept is as it is. Perhaps we should use token instead of token

Reply:
In RATS parlance there is no "raw" evidence, just evidence.  So I am not sure what is the distinction you are trying to make here.  Also, "token" is an Arm-specific term that we have used for CCA and PSA but does not necessarily fit the wider RATS context.

Reply:
maybe better saying: "[...] sending attestation messages over HTTP(S) [...]" because the EAT collection could be used to encapsulate attestation results too.

Reply:
yes, this makes more sense,  I will change this!

Reply:
But notice the word token already used in the draft 3 times!!!

Reply:
And remember EAT stands for "Entity Attestation Token" so nothing wrong in using the word, token!

Reply:
Sure, no one will be harmed by the use of the t-word :)

However, IMO all new documents should make the effort of aligning with RATS terminology.